content/applications/finance/accounting/taxation/fiscal_year/close_fiscal_year.rst

```diff
@@ -2,92 +2,92 @@
+  dashboard, validating any open :guilabel:`Vendor Bills` and :guilabel:`Customer Invoices` with
+  their payments. This step is optional, however it may assist the year-end process if all
+  outstanding payments and invoices are reconciled, and could lead finding errors or mistakes in the
+  system.
 
-- Run the optional **Payments Matching** feature, under the **More**
-  dropdown on the dashboard, validating any open **Vendor Bills** and
-  **Customer Invoices** with their payments. This step is optional,
-  however it may assist the year-end process if all outstanding
-  payments and invoices are reconciled, and could lead finding
-  errors or mistakes in the system.
+- Your accountant/bookkeeper will likely verify your balance sheet items and book entries for:
 
-- Your accountant/bookkeeper will likely verify your balance sheet
-  items and book entries for:
+ - Year-end manual adjustments.
 
-  - Year-end manual adjustments, using the **Adviser Journal Entries**
-    menu (For example, the **Current Year Earnings** and **Retained
-    Earnings** reports).
+ - **Work in Progress**.
 
-  - **Work in Progress**.
+ - **Depreciation Journal Entries**.
 
-  - **Depreciation Journal Entries**.
+ - **Loans**.
 
-  - **Loans**.
+ - **Tax adjustments**.
 
-  - **Tax adjustments**.
+If your accountant/bookkeeper is going through end of the year auditing, they may want to have paper
+copies of all balance sheet items (such as loans, bank accounts, prepayments, sales tax statements,
+etc...) to compare these with your Odoo balances.
 
-If your accountant/bookkeeper is going through end of the year auditing,
-they may want to have paper copies of all balance sheet items (such as
-loans, bank accounts, prepayments, sales tax statements, etc...) to
-agree these against your Odoo balances.
+.. important::
+  During this process, it is good practice to set the :guilabel:`Lock date for Non-Advisers` to the
+  last day of the preceding financial year, which is set under the accounting configuration. This
+  way, the accountant can be confident that nobody is changing the previous year transactions while
+  auditing the books.
 
-During this process, it is good practice to set the **Lock date for
-Non-Advisers** to the last day of the preceding financial year, which is
-set under the accounting configuration. This way, the accountant can be
-confident that nobody is changing the previous year transactions
-while auditing the books.
+.. warning::
+  On this screen, you also have a field :guilabel:`all users Lock date`. This lock date will
+  prevent journal entries creation prior to the defined date for all users.
+  This lock date is irreversible and cannot be removed.
 
-.. image:: close_fiscal_year/close_fiscal_year01.png
-   :align: center
+.. note::
+  The :guilabel:`all journal Lock date` prevents journal entry creation or modification prior the
+  defined date except for advisor users.
+  This lock date is reversible and can be removed.
```

Comment:
I can't find this field, and the description seems very similar to the **lock date for non-advisers** field, right? I removed it for now.